src/Phinx/Console/Command/Create.php

```diff
@@ -52,6 +58,13 @@ protected function configure()
                 PHP_EOL,
                 PHP_EOL
             ));
+
+        // An alternative template.
+        $this->addOption('template', 't', InputOption::VALUE_REQUIRED, 'Use an alternative template');
+
+        // A classname to be used to gain access to the template content as well as the ability to
+        // have a callback once the migration file has been created.
+        $this->addOption('class', 'l', InputOption::VALUE_REQUIRED, 'Use a class implementing "' . self::CREATION_INTERFACE . '" to generate the template');
```

Comment:
This message should not use `CREATION_INTERFACE` constant, as it could be overloaded. Just use `"CreationInterface"`.